src/integration-tests/introspector/wl-introspect-pod.yamlt

```diff
@@ -17,6 +17,12 @@ spec:
     env:
     - name: JAVA_OPTIONS
       value: "-Djava.security.egd=file:/dev/./urandom "
+    - name: USER_MEM_ARGS
+      value: "-XX:MaxRAMFraction=1 -XX:+UseContainerSupport "
+    - name: NODEMGR_JAVA_OPTIONS
+      value: "-Dnodemgr.java.options -Djava.security.egd=file:/dev/./urandom "
+    - name: NODEMGR_MEM_ARGS
+      value: "-Xms64m -Xmx100m -XX:+UseContainerSupport "
```

Comment:
Should we use NODEMGR_MEM_ARGS value other than the default values for this test? Also, Ryan has recently removed "-XX:+UseContainerSupport" in PR 1368